cython/_pocketsphinx.pyx

```diff
@@ -1535,6 +1556,39 @@ cdef class Vad:
 
 cdef class Endpointer:
     """Simple endpointer using voice activity detection.
+
+    Args:
+      window(float): Length in seconds of window for decision.
+      ratio(float): Fraction of window that must be speech or
+                    non-speech to make a transition.
+      mode(int): Aggressiveness of voice activity detction (0-3)
+      sample_rate(int): Sampling rate of input, default is 16000.
+                        Rates other than 8000, 16000, 32000, 48000
+                        are only approximately supported, see note
+                        in `frame_length`.  Outlandish sampling
+                        rates like 3924 and 115200 will raise a
+                        `ValueError`.
+      frame_length(float): Desired input frame length in seconds,
+                           default is 0.03.  The *actual* frame
+                           length may be different if an
+                           approximately supported sampling rate is
+                           requested.  You must *always* use the
+                           `frame_bytes` and `frame_length`
+                           attributes to determine the input size.
+
+    Attributes:
+      sample_rate(int): Sampling rate of input (default is 16000)
+      frame_bytes(int): Number of bytes in a frame accepted by `process`.
+      frame_length(float): Length of a frame (*may be different from
+                           the one requested in the constructor*!)
+      in_speech(boolean): Are we currently in a speech region?
+      speech_start(float): Start of previous speech segment.
+      speech_end(float): End of previous speech segment.
+
+    Raises:
+      ValueError: Invalid input parameter.  Also raised if the ratio
+                  makes it impossible to do endpointing (i.e. it
+                  is more than N-1 or less than 1 frame).
     """
```

Comment:
doc strings are much appreciated!

Reply:
They still need some work... but you can also see them here: http://pocketsphinx5.readthedocs.io/

(upload to PyPI of the module itself is forthcoming)